### PR TITLE
doc: update reflection-milestone2.md

### PR DESCRIPTION
**Pull request by jennifer-hoang**
Hello all,

Here is my draft for our reflection. Please let me know what you think, and if you have any additional thoughts you would like to add.

Thanks!

### CODE REVIEW

**Review by michelle-wms — APPROVED** [review]
looks good to me!

**Review by christopheralex — APPROVED** [review]
Looks good :) Great work Jennifer

File: doc/reflection-milestone2.md [hunk 1]
```diff
@@ -1,3 +1,29 @@
...
+
+Overall, our dashboard is able to provide a quick glimpse at the music trends across artists and genres on Spotify in 2020.
+
+### Future Improvements 
+To improve the usability of our dashboard, our next steps are to include more interactivity and linkage between plots.
+A feature that we would like to implement is to allow users to select an artist from the "Top Artists" chart and link that
+selection to the "Artist Trend" and "Artist Popularity Record" plots.
+
+Additionally, there are improvements that could be made to the overall layout and user interface design. 
+The sizing of plots and the font size of the axes could be increased to improve the aesthetics and readability.
```
Comment (xiangwxt) [inline doc/reflection-milestone2.md:22]: Maybe we can improve these within the milestone 2

File: doc/reflection-milestone2.md [hunk 2]
```diff
@@ -1,3 +1,29 @@
...
+The 3 plots in the "Artists/Genre" tab allow users to explore top artists, artist popularity trends over various album releases,
+and the overall popularity distribution of their songs. We have implemented two widgets on this page, 
+which allow users to select the genre for the top artists plot from a dropdown menu.
+Users can also select specific artists of interest from a dropdown menu to explore their trend and distribution plots in detail.
+
+In the second "Song Characteristics" tab, users can compare musical features between popular and non-popular songs.
+The musical feature displayed on the X-axis, as well as the genre of music, can be selected from the dropdown menu.
+
+Overall, our dashboard is able to provide a quick glimpse at the music trends across artists and genres on Spotify in 2020.
+
```
Comment (xiangwxt) [inline doc/reflection-milestone2.md:15]: This suffices the need of our target users, content managers at Spotify, to get music tracks insights visually and quickly to support their data driven decision making process.

**Review by xiangwxt — APPROVED** [review]
Hi Jennifer, the reflection is quite comprehensive of what we have done, good job. I just have a few comments and suggestions. Hope those would be helpful.

**Comment by christopheralex** [comment]
I am sorry I merged this PR . Would wait for everyone's approval next time.
I can make a github issue of this for Jenn and ourselves to refer.
Thanks you for you review Thea Wenxin

**Pull request merged**